Export Analysis Reports/Sdl.Community.ExportAnalysisReports/ReportExporterControl.cs

```diff
@@ -5,65 +5,79 @@
 using System.IO;
 using System.Linq;
 using System.Text;
-using System.Threading.Tasks;
 using System.Windows.Forms;
 using System.Xml;
 using Sdl.Community.ExportAnalysisReports.Helpers;
 using Sdl.Community.ExportAnalysisReports.Interfaces;
 using Sdl.Community.ExportAnalysisReports.Model;
 using Sdl.Community.ExportAnalysisReports.Service;
 using static System.String;
-using Help = Sdl.Community.ExportAnalysisReports.Helpers.Help;
 
 namespace Sdl.Community.ExportAnalysisReports
 {
 	public partial class ReportExporterControl : Form
```

Comment:
This class has to be abstracted better... by breaking it into more classes with their corresponding interfaces. Don't forget about SOLID.

Reply:
I've moved methods within the ReportService and StudioService